jackson-databind vulnerability

On our CI system, a CVE scan detected a vulnerability due to the shaded com.fasterxml.jackson.core version pulled in by jackson-1.0.0-beta7.

`[ERROR] jackson-1.0.0-beta7.jar/META-INF/maven/com.fasterxml.jackson.core/jackson-databind/pom.xml: CVE-2020-25649`

The issue seems to have been fixed in jackson-databind already as of milestone 2.9.10.7 (see https://github.com/FasterXML/jackson-databind/issues/2589).

Version Information
Deeplearning4j version: 1.0.0-beta7
Platform information: Linux/Mac OS

--- a/pom.xml
+++ b/pom.xml
@@ -228,14 +228,14 @@
         <archunit.version>0.14.1</archunit.version>
         <logback.version>1.2.3</logback.version>
         <jackson.version>2.10.1</jackson.version>
-        <jackson.databind.version>2.10.3</jackson.databind.version>
+        <jackson.databind.version>2.9.10.8</jackson.databind.version>
         <shaded.snakeyaml.version>1.24</shaded.snakeyaml.version>
         <geo.jackson.version>2.8.7</geo.jackson.version>
         <lombok.version>1.18.16</lombok.version>
         <cleartk.version>2.0.0</cleartk.version>
         <lucene-solr.version>7.7.1</lucene-solr.version>
         <json.versiounitn>20131018</json.versiounitn>
-        <google.protobuf.version>3.8.0</google.protobuf.version>
+        <google.protobuf.version>3.12.2</google.protobuf.version>
         <google.protobuf.solr.version>2.6.1</google.protobuf.solr.version>
         <failIfNoTests>false</failIfNoTests>
         <!-- Hadoop version used by Spark 1.6.3 and 2.2.1 (and likely others) -->
@@ -248,7 +248,7 @@
         <!-- 2.7.3 has Scala 2.11, 2.12 and 2.13 support only (no 2.10) -->
         <playframework.version>2.7.3</playframework.version>
         <javax.ws.rs.version>2.0</javax.ws.rs.version>
-        <guava.jre.version>28.0-jre</guava.jre.version>
+        <guava.jre.version>30.1.1-jre</guava.jre.version>
         <guava.android.version>28.0-android</guava.android.version>
         <gson.version>2.8.0</gson.version>
         <fbs.version>1.2.0-3f79e055</fbs.version>

--- a/pom.xml
+++ b/pom.xml
@@ -227,8 +227,8 @@
         <junit.version>5.8.0-M1</junit.version>
         <archunit.version>0.14.1</archunit.version>
         <logback.version>1.2.3</logback.version>
-        <jackson.version>2.10.1</jackson.version>
-        <jackson.databind.version>2.9.10.8</jackson.databind.version>
+        <jackson.version>2.12.2</jackson.version>
+        <jackson.databind.version>2.12.2</jackson.databind.version>
         <shaded.snakeyaml.version>1.24</shaded.snakeyaml.version>
         <geo.jackson.version>2.8.7</geo.jackson.version>
         <lombok.version>1.18.16</lombok.version>

